### PR TITLE
Create mar10.wsgidav.yaml

### PR DESCRIPTION
**Pull request by mar10**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40996)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
```
Build log #L28
 Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\m\mar10\wgidav\4.0.1.0', expected 'manifests\m\mar10\WsgiDAV\4.0.1.0'
```

File: manifests/m/mar10/wsgidav/4.0.1.0/mar10.wsgidav.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - wsgi
+  - webdav
+  - application
+  - server
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/mar10/wsgidav/releases/download/v4.0.1/WsgiDAV-4.0.1.0-win64.msi
+  InstallerSha256: 385027629F33BE16ECD303332B5BA952F64350C690566888DCDF8A54ECA77C52
+  ProductCode: "{50CAACE8-A9F0-40DB-B10F-4E69CA43D531}"
```
Comment (ItzLevvie) [inline manifests/m/mar10/wsgidav/4.0.1.0/mar10.wsgidav.yaml:21]:
```suggestion
  ProductCode: "{0E819071-6181-4E2A-BEF2-43F6C757813D}"
```

File: manifests/m/mar10/wsgidav/4.0.1.0/mar10.wsgidav.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ShortDescription: Generic and extendable WebDAV server based on WSGI
+Tags: 
+  - web
+  - wsgi
+  - webdav
+  - application
+  - server
+Installers:
+- Architecture: x64
+  InstallerType: msi
```
Comment (ItzLevvie) [inline manifests/m/mar10/wsgidav/4.0.1.0/mar10.wsgidav.yaml:18]:
```suggestion
  InstallerType: msi
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/mar10/wsgidav/4.0.1.0/mar10.wsgidav.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/m/mar10/wsgidav/4.0.1.0/mar10.wsgidav.yaml:21]: image

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**